### DISCUSSION TITLE
Admin area missing

### DISCUSSION THREAD

**Discussion by Chaffy-0** [roxy-wi/roxy-wi#269] [Q&A] [answered]
I'm going through the "How to setup servers, group and SSH credentials" how to doc and I don't have an Admin area to create the new group.  I still have the original admin account that I changed the default password from and I've created a new admin account but neither are showing the Admin area.

**Comment by Aidaho12** [comment]
Could you show print screen of your admin area page

**Reply by Chaffy-0** [reply]
![image](https://user-images.githubusercontent.com/52983620/114052130-c33e0680-985b-11eb-8714-79d70008ab21.png)

Shouldn't the admin area menu be just below the servers menu on the left side?

**Comment by Aidaho12** [comment]
It looks like you do not has a supeAadmin role, but just admin

**Reply by Chaffy-0** [reply]
I don't see supeadmin under the different roles to apply to an account.  Is there a way to reset the users and groups to default without reinstalling?  I don't mind reinstalling if that's what it takes.

**Comment by Aidaho12** [comment, marked as answer]
Yes, delete the db and create it again

**Reply by Chaffy-0** [reply]
That worked, thank you!!
